functions/src/expressRestApp/routes/route.service.ts

```diff
@@ -0,0 +1,18 @@
+import express from "express";
+import { getData } from "../services/stations.service";
+
+const router = express.Router();
+

```

Comment:
please add handler for base url /stations
`router.get("/", async (req: Request, res: Response) => {
  res.status(200).json("Stations info currently is unavalible");
});`